modules/ocf_stats/files/munin/mail-munin-alert

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+"""Accepts alerting information from Munin on the command line and forwards the
+alerts to root.
+
+The expected call format from Munin is
+
+    mail-munin-alerts <hostname> [<list of warnings and criticals>]
+
+The list of warnings/criticals allows filtering of unhelpful messages only
+containing OKs or UNKNOWNs.
+
+Munin seems to require duplicating the name of the script as the first argument
+for some reason; this is stripped out.
+"""
+from sys import argv
+from sys import stdin
+
+from ocflib.constants import MAIL_ROOT
+from ocflib.misc.mail import send_mail
+
+
+MAIL_FROM = 'Munin <munin@ocf.berkeley.edu>'
+
+
+def main(args):
+    subject = 'Munin alert for {}'.format(args[1])
+    body = ''
+    for line in stdin:
+        body += line
+    send_mail(MAIL_ROOT, subject, body, MAIL_FROM)
+
+
+if __name__ == '__main__':
+    if len(argv) < 2:
+        raise RuntimeError(
+            'Expected command-line arguments; possible misconfiguration')
+    if argv[1] == argv[0]:
+        args = argv[1:]
+    else:
+        args = argv
```

Comment:
The doc comment reads

> Munin seems to require duplicating the name of the script as the first argument
for some reason; this is stripped out.

I'll add a comment here as well.